frontend/src/metabase-types/api/mocks/actions.ts

```diff
@@ -77,21 +77,22 @@ export const createMockImplicitQueryAction = ({
 
 export const createMockImplicitCUDActions = (
   modelId: CardId,
+  startId: number = 0,
```

Comment:
Is this parameter added to avoid creating actions with the existing ID e.g. there are other query actions with ID 1, 2, and 3 already. If that's the case do you think the test utility should commodate that? Like automatically increments ID just like how databases do it by default, but still allows you to specify ID if needed.

Reply:
yes, definitely. faker.js does it